Source/WebCore/platform/graphics/cocoa/FontFamilySpecificationCoreText.cpp

```diff
@@ -46,8 +47,8 @@ FontFamilySpecificationCoreText::~FontFamilySpecificationCoreText() = default;
 
 FontRanges FontFamilySpecificationCoreText::fontRanges(const FontDescription& fontDescription) const
 {
-    auto& fontPlatformData = FontFamilySpecificationCoreTextCache::forCurrentThread().ensure(FontFamilySpecificationKey(m_fontDescriptor.get(), fontDescription), [&] () {
-        auto size = fontDescription.computedSize();
+    auto size = fontDescription.computedSize();
+    FontPlatformData platformData = FontFamilySpecificationCoreTextCache::forCurrentThread().ensure(FontFamilySpecificationKey(m_fontDescriptor.get(), fontDescription), [&] () {
```

Comment:
Fix the formatting here to `[&]()` without the space, please.

Reply:
(FYI https://webkit.org/code-style-guidelines/#spacing-lambda-paren)

Reply:
Previously it says `auto& fontPlatformData` and now it says `FontPlatformData platformData`

1. I think it's better to use `auto` because the name tells you the type
2. Making an unconditional copy of the FontPlatformData is unfortunate. It's sizable.